### PR TITLE
Fix mobile width/ratio issue on all blog post YouTube video iframe embeds

### LINKED ISSUE #12234: Fix mobile width/ratio issue on all blog post YouTube video iframe embeds
# Bug report

## Describe the bug

On mobile view, youtube iframes are forcing the site layout to be too wide when not using corrective classNames or using the `<div className="video-container">` hack [like this](https://github.com/supabase/supabase/blob/2579f1339b3eb855a20aaa225bb930d95bf80ffe/apps/www/_blog/2023-02-07-chatgpt-supabase-docs.mdx?plain=1#L57-L65)

To resolve this issue, update all the youtube iframes in `/apps/www/_blog/*.mdx` so they display with consistent size ratio and also 100% of the container they are in.

### PR DESCRIPTION
**Pull request by aidenkwong**
## What kind of change does this PR introduce?

Bug fix

## What is the current behavior?

Please link any relevant issues here. 

#12234 

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/www/_blog/2020-06-01-supabase-alpha-may-2020.mdx [hunk 1]
```diff
@@ -14,15 +14,17 @@ video: https://www.youtube.com/v/e4qXmcEFaUs
...
-  height="350"
-  src="https://www.youtube-nocookie.com/embed/e4qXmcEFaUs"
-  frameBorder="0"
-  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-  allowfullscreen
-></iframe>
+<div className="video-container">
+  <iframe
+    className="w-full"
+    width="700"
```
Comment (MildTomato) [inline apps/www/_blog/2020-06-01-supabase-alpha-may-2020.mdx:20]: I think we can remove width and height from everything now?

**Review by MildTomato — CHANGES_REQUESTED** [review]
This is great 🚀 maybe a small change to remove some width and heights though. see comment below

File: apps/www/_blog/2020-06-01-supabase-alpha-may-2020.mdx (hunk 1 above)
Comment (aidenkwong) [reply apps/www/_blog/2020-06-01-supabase-alpha-may-2020.mdx]: Just removed

**Comment by kiwicopple** [comment]
there appears to be a build issue here:

```
/blog/building-a-realtime-trello-board-with-supabase-and-angular
/blog/supabase-js-v2
```

**Comment by kiwicopple** [comment]
thanks for this - this one was picked up by another contributor here with all the necessary fixes:

https://github.com/supabase/supabase/pull/12418

@aidenkwong - feel free to reach out to me on twitter if you want some swag !

**Pull request closed without merging**